cypress/support/metamask/selectors.ts

```diff
@@ -0,0 +1,40 @@
+
+export class Selectors {
```

Comment:
**question** if metamask updates selectors, will we also need to update them here?

Reply:
Yeah, so this is where things come undone a bit (in my opinion anyway). I want to look at a more sustainable way to do this, because right now this is very brittle to changes to metamask, and very finnicky in general. In reality, I want to be able to just say like `getButtonThatHasStartHereText()` or soemthing to that effect. We can probably get there, but will have to do some digging. This is a good start to hack it to work though I think